plugins/Compressor/CompressorControlDialog.h

```diff
@@ -108,6 +108,12 @@ private slots:
 	void drawKneePixmap2();
 	void drawMiscPixmap();
 	void drawGraph();
+	void mouseDoubleClickEvent(QMouseEvent *event) override;
+	inline bool getGuiVisibility()
+	{
+		return guiVisibility;
+	}
```

Comment:
```suggestion
```
I would just remove this method since it's only used once and from within this class.